pkg/config/process.go

```diff
@@ -0,0 +1,50 @@
+package config
+
+import (
+	"os"
+	"time"
+)
+
+func setupProcesses(config Config) {
+	config.SetDefault("process_config.enabled", "false")
+	// process_config.enabled is only used on Windows by the core agent to start the process agent service.
+	// it can be set from file, but not from env. Override it with value from DD_PROCESS_AGENT_ENABLED.
+	ddProcessAgentEnabled, found := os.LookupEnv("DD_PROCESS_AGENT_ENABLED")
+	if found {
+		AddOverride("process_config.enabled", ddProcessAgentEnabled)
+	}
+
+	config.BindEnv("process_config.process_dd_url", "")
+
+	config.SetKnown("process_config.dd_agent_env")
+	config.SetKnown("process_config.enabled")
+	config.SetKnown("process_config.intervals.process_realtime")
+	config.SetKnown("process_config.queue_size")
+	config.SetKnown("process_config.rt_queue_size")
+	config.SetKnown("process_config.max_per_message")
+	config.SetKnown("process_config.max_ctr_procs_per_message")
+	config.SetKnown("process_config.cmd_port")
+	config.SetKnown("process_config.intervals.process")
+	config.SetKnown("process_config.blacklist_patterns")
+	config.SetKnown("process_config.intervals.container")
+	config.SetKnown("process_config.intervals.container_realtime")
+	config.BindEnvAndSetDefault("process_config.dd_agent_bin", defaultDDAgentBin, "DD_PROCESS_AGENT_DD_AGENT_BIN", "DD_PROCESS_CONFIG_DD_AGENT_BIN")
```

Comment:
Should probably keep a consistent environment variable priority between `DD_PROCESS_AGENT` and `DD_PROCESS_CONFIG` prefixes